README.md

```diff
@@ -17,17 +19,45 @@
 
 # ormb
 
-Managing Machine Learning/Deep Learning Models like Docker Images.
+ormb is an open-source model registry to manage machine learning model. 
 
 ormb helps you manage your Machine Learning/Deep Learning models. It makes your models easy to create, version, share and publish.
 
-ormb can publish models by reading the information from a `ormbfile.yaml`. A `ormbfile.yaml` is a text document that contains all the information about the model.
-
 ## Installation
 
-```bash
-go install github.com/caicloud/ormb
+You can install the pre-compiled binary, or compile from source.
+
+### Install the pre-compiled binary
+
+Download the pre-compiled binaries from [the releases](https://github.com/caicloud/ormb/releases) page and copy to the desired location.
+
+### Compile from source
```

Comment:
Do it need to provide docker image in the future.

Reply:
Oh yeah, let's open an issue for it.

Reply:
We should not use images in caicloud internal registry.

Reply:
emmm, SGTM

Reply:
https://github.com/caicloud/ormb/issues/54